BUILD.bazel

```diff
@@ -2587,17 +2587,11 @@ alias(
     visibility = ["//visibility:public"],
 )
 
-filegroup(
-    name = "experimental_py_proto",
-    srcs = [
-        "//src/ray/protobuf:autoscaler_py_proto",
-    ],
-)
-
 filegroup(
     name = "all_py_proto",
     srcs = [
         "//src/ray/protobuf:agent_manager_py_proto",
+        "//src/ray/protobuf:autoscaler_py_proto",
```

Comment:
This is actually much easier than what I did initially (adding another build target which put things under `ray/core/generated/experimental`)